app/views/DataEntryView.js

```diff
@@ -0,0 +1,40 @@
+
+var Template = require('../util/Template');
+var TelemetryData = require('../models/TelemetryData')
+var fs = require('fs');
+var Logger = require('../util/Logger');
+var app_config=require('./config/application_config.js');
+
+
+module.exports = function (Marionette) {
+
+    return Marionette.ItemView.extend({
+        template: Template('DataEntryView'), 
+        className: 'gainsAdjustView', 
+
+        ui: {
+            name_field: "#fileName",
+            comment_field: "#comment",
+            save_button: "#saveComment"
+     
+        },
+      
+        events: {
+            "click #saveComment": "saveData",
+            
```

Comment:
Extra spaces means less code shows in a window pane, and (in this case) adds no additional clarity/logical separation.